app/src/main/kotlin/com/wire/android/ui/debug/DebugDataOptions.kt

```diff
@@ -434,6 +460,27 @@ private fun DebugToolsOptions(
                 )
             }
         )
+
+        RowItemTemplate(
+            modifier = Modifier.wrapContentWidth(),
+            title = {
+                Text(
+                    style = MaterialTheme.wireTypography.body01,
+                    color = MaterialTheme.wireColorScheme.onBackground,
+                    text = stringResource(R.string.debug_settings_register_fcm_push_token),
+                    modifier = Modifier.padding(start = dimensions().spacing8x)
+                )
+            },
+            actions = {
+                WirePrimaryButton(
+                    minSize = MaterialTheme.wireDimensions.buttonMediumMinSize,
+                    minClickableSize = MaterialTheme.wireDimensions.buttonMinClickableSize,
+                    onClick = onResendFCMToken,
+                    text = stringResource(R.string.debug_settings_force_api_versioning_update_button_text),
+                    fillMaxWidth = false
+                )
+            }
+        )
```

Comment:
request: can we for now please keep the button for internal usage only for internal builds you can check this by calling `BuildConfig.PRIVATE_BUILD`